CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 4.1.4
+### API Changes
+* Deprecate `Transaction` map properties `revenueCatId` and `productId`. They will be substituted with `transactionIdentifier` and `productIdentifier` respectively (#211)
```

Comment:
This wasn't added to the `CHANGELOG.md` automatically, I added it after since it was tagged as `refactor` and so it didn't appear in the changelog.

Reply:
Ahh you're right! Yeah, will do so thanks!